### PR TITLE
Added Zoe to organizer list

### PR DESCRIPTION
**Pull request by TheDataLeek**

### CODE REVIEW

File: application/readme.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Submission Design Thoughts
+
+> Ian
```
Comment (svvitale) [inline application/readme.md]: Did you mean to include this?

Comment (TheDataLeek) [reply application/readme.md]: Ehhhhhh it was in the other branch that i never merged in.

It doesn't really fit this PR but it's valuable content that should probably get included anyway.

Comment (svvitale) [reply application/readme.md]: I just worry that Ian may not want that quote publicized (even though our members understand that there's some amount of curation required in the talk selection process).

Comment (TheDataLeek) [reply application/readme.md]: Good point, i'll prune it

**Review by svvitale — APPROVED** [review]

**Pull request merged**